src/libraries/Microsoft.Extensions.Logging.EventSource/tests/EventSourceLoggerTest.cs

```diff
@@ -720,7 +810,7 @@ private static class EventTypes
             { "E6MSG", (e) => VerifySingleEvent(e, "Logger2", EventTypes.Message, 6, null, LogLevel.Warning) },
 
             { "E7FM", (e) => VerifySingleEvent(e, "Logger3", EventTypes.FormattedMessage, 7, null, LogLevel.Information,
-                @"""FormattedMessage"":""Logger3 Event7 Information inner scope closed " + DoubleParam2.ToString() + " 37") },
+                @"""FormattedMessage"":""Logger3 Event7 Information inner scope closed " + DoubleParam2.ToString("G", CultureInfo.InvariantCulture) + " 37") },
             { "E7JS", (e) => VerifySingleEvent(e, "Logger3", EventTypes.MessageJson, 7, null, LogLevel.Information,
                         @"""ArgumentsJson"":{""stringParam"":""inner scope closed"",""doubleParam"":""" + DoubleParam2.ToString() + @""",""intParam"":""37""") },
```

Comment:
nit: I see another double ToString() here. Should that be fixed here as well? If it's not causing failures it might be because it's not used, in which case why is it present?

Reply:
I noticed them as well. but only this change should be sufficient

Reply:
I’d fix them all for correctness.  It’s not like we have a reason for using current culture

Reply:
I'll do that in a future PR.